packages/gluestick/src/commands/test/test.js

```diff
@@ -38,7 +38,7 @@ const getJestDefaultConfig = (aliases: Object, webpackRules: RegExp[]): string[]
 
   // Handling Static Assets = mock them out
   const fileRegex: RegExp = new RegExp(
-    `${webpackRules[3].source}|${webpackRules[4].source}|${webpackRules[5].source}`,
+    `${webpackRules[3].source}|${webpackRules[4].source}`,
```

Comment:
This still can be `undefined` if the user overrides those rules. I'd go with hardcoding this:

```
"moduleNameMapper": {
  "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
  "\\.(css|scss|less)$": "<rootDir>/__mocks__/styleMock.js"
}
```

If then they user adds something else that needs to be handled, they can add that case in `package.json` under `jest`.